app/models/permit_collaboration.rb

```diff
@@ -0,0 +1,25 @@
+class PermitCollaboration < ApplicationRecord
+  belongs_to :collaborator
+  belongs_to :permit_application
+
+  enum collaboration_type: { submission: 0, review: 1 }
+  enum collaborator_type: { delegatee: 0, assignee: 1 }, _default: 0
```

Comment:
delegate is probably a better word than delagatee

Reply:
In regular parlance, a delegate refers to a person chosen or elected to represent a group, organization, or constituency, typically at a conference, convention, or other formal meeting. Delegates are given the authority to act, speak, or vote on behalf of the people they represent. The term can also refer to someone assigned specific responsibilities or tasks by someone in a position of authority.

Reply:
@bkmaibach seems like a synonymn https://www.merriam-webster.com/thesaurus/delegatee